Makefile

```diff
@@ -151,6 +151,14 @@ check-docker-buildx-tool-check:
 		exit 1; \
 	fi
 
+# Check that docker compose is installed.
+.PHONY: check-docker-compose-tool-check
+check-docker-compose-tool-check:
+	@if ! docker compose >/dev/null 2>&1; then \
```

Comment:
should this be $(CONTAINER) compose?

Reply:
True, I adjusted it and also fixed it for the `buildx` check.